server/app/index.js

```diff
@@ -11,7 +11,7 @@ module.exports = function (db) {
 
     // Routes that will be accessed via AJAX should be prepended with
     // /api so they are isolated from our GET /* wildcard.
-    app.use('/api', require('./routes'));
+    app.use('/', require('./routes/'));
```

Comment:
Yeah that makes plenty of sense, I was just commenting on the '/' after the word routes on line 14. And also the fact that this PR doesn't seem to be in a branch from master?